### DISCUSSION TITLE
創作平台ArtStation不完全禁止AI創作

### DISCUSSION THREAD

**Discussion by Xuewen628** [1101-GS3073-C/Python-Community-GS3073-C#636] [General]
最近在網路上常看到AI繪圖的蹤影
我有思考過若AI創作被拿來當作營利用途
那對藝術創作者也算是一種褻瀆嗎？
換個角度 作品的隱私權也痕難獲得完全的保證

**Comment by awehao** [comment]
AI 深度學習研究領域近年擴展到創作圈，衍生出許多爭議。先前藝術設計創作平台 ArtStation 並沒有制定 AI 相關規章，許多創作者憤而發起「拒絕 AI 繪圖」（NO TO AI GENERATED IMAGES）抗議。雖然 ArtStation 於 15 日發布新規章，但終究沒有完全禁止 AI 創作，讓用戶們感到相當失望。
ArtStation 主要是 2D、3D 繪圖、影像等創作領域專業人士使用的平台，許多自由業者、業餘人士會透過 ArtStation 接受個人委託或商案。在 AI 圖像創作在全球如雨後春筍般出現後，不少創作者擔憂大量的 AI 創作會佔據創作平台的趨勢頁面，進而降低真人創作者作品的曝光度。

**Comment by awkward-willy** [comment]
我認為目前主要的問題出在 AI 在訓練時的資料集可能有侵權等嫌疑，或者作者本人並不清楚自己的作品是否有作為學習的一部分。進而導致若產生很類似，或甚至相同的圖案時，作家心中難免會產生不平的情緒。而且，若 AI 能享有營利的福利，侵權時也應當付出相對應的義務與責任。

**Comment by larry40305** [comment]
現在這項技術才剛起步，相關的法律必須要快點跟上以保護各類創作者
